index.html

```diff
@@ -8,22 +8,146 @@
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <div class="button" >
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+      </div>
+    </div>
+    <div id="project-section3" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div id="project-section5" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div  id="project-section7" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div id="project-section9" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""> </button>
+      </div>
+    </div>
+    <div id="project-section11" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    </div>
+  </section>
+  <section id="about-section">
+  <div id="about-sction-div">
+    <h2>About</h2>
+    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+       Labore praesentium, ipsa necessitatibus maioresquaeiusto eveniet
```

Comment:
- [ ] Your font should be similar to the one provided in the design. Try to match the one provided in the Figma design.  Try to improve your design based on provided design. 


Figma design | Your design
--- | ---
![image](https://user-images.githubusercontent.com/43877689/170815590-57adf599-aadc-4643-98f3-efa7c87e7698.png) | ![image](https://user-images.githubusercontent.com/43877689/170815611-d41031f1-7ee1-4c3f-b8ad-95571dfd3587.png)

Reply:
Done